### ISSUE TITLE
Fix Accordion React component styling for light mode

### ISSUE THREAD

**Issue by Devon-White** [signalwire/docs-legacy#228]
Fails accessibility right now

**Comment by Devon-White** [comment]
Fixed in latest PR
